.Rbuildignore

```diff
@@ -2,3 +2,4 @@ fitRotavirus.R
 ^.*\.Rproj$
 ^\.Rproj\.user$
 ^\.travis.yml
+^\.travis\.yml$
```

Comment:
This line should probably just replace the one above? (They're both about `.travis.yml`)

Reply:
Indeed it should!